### DISCUSSION TITLE
Dark mode for documentation page

### DISCUSSION THREAD

**Discussion by suspiciousRaccoon** [bigskysoftware/htmx#2573] [Ideas]
Docs page are too bright, dark mode would be nice c:

**Comment by Telroshan** [comment]
Then you should be pleased to know that [a PR](https://github.com/bigskysoftware/htmx/pull/2562) was recently merged to add dark mode to the website 😄 
It will be released along htmx 2, but you can already see how it looks at https://dev.htmx.org/
